ext/standard/basic_functions.stub.php

```diff
@@ -652,41 +652,41 @@ function strip_tags(string $str, array|string|null $allowable_tags = null): stri
 function setlocale(int $category, $locales, ...$rest): string|false {}
 
 /** @param array $result */
-function parse_str(string $encoded_string, &$result): void {}
+function parse_str(string $string, &$result): void {}
 
 function str_getcsv(string $string, string $delimiter = ",", string $enclosure = "\"", string $escape = '\\'): array {}
 
-function str_repeat(string $input, int $mult): string {}
+function str_repeat(string $string, int $times): string {}
 
-function count_chars(string $input, int $mode = 0): array|string {}
+function count_chars(string $string, int $mode = 0): array|string {}
 
-function strnatcmp(string $s1, string $s2): int {}
+function strnatcmp(string $string1, string $string2): int {}
 
 function localeconv(): array {}
 
-function strnatcasecmp(string $s1, string $s2): int {}
+function strnatcasecmp(string $string1, string $string2): int {}
 
 function substr_count(string $haystack, string $needle, int $offset = 0, ?int $length = null): int {}
 
-function str_pad(string $input, int $pad_length, string $pad_string = " ", int $pad_type = STR_PAD_RIGHT): string {}
+function str_pad(string $string, int $length, string $pad_string = " ", int $pad_type = STR_PAD_RIGHT): string {}
 
-function sscanf(string $str, string $format, mixed &...$vars): array|int|null {}
+function sscanf(string $string, string $format, mixed &...$vars): array|int|null {}
 
-function str_rot13(string $str): string {}
+function str_rot13(string $string): string {}
 
-function str_shuffle(string $str): string {}
+function str_shuffle(string $string): string {}
 
-function str_word_count(string $str, int $format = 0, ?string $charlist = null): array|int {}
+function str_word_count(string $string, int $format = 0, ?string $characters = null): array|int {}
 
-function str_split(string $str, int $split_length = 1): array {}
+function str_split(string $string, int $length = 1): array {}
 
-function strpbrk(string $haystack, string $char_list): string|false {}
+function strpbrk(string $string, string $characters): string|false {}
 
-function substr_compare(string $main_str, string $str, int $offset, ?int $length = null, bool $case_insensitivity = false): int {}
+function substr_compare(string $main_string, string $string, int $offset, ?int $length = null, bool $case_insensitive = false): int {}
```

Comment:
Bob's suggestions were `$haystack` and `$needle`, and I also pretty much love these names.

Reply:
And that's all I could check today.

Reply:
Makes sense, I went with it.